src/app/geo-header/geo-header.component.ts

```diff
@@ -28,4 +28,23 @@ export class GeoHeaderComponent {
       .dispatch(changeSearchFromAddress($value));
   }
 
+  geolocate() {
+    this.store
```

Comment:
At this moment it is not consumed elsewhere. But it will be. It is not a state that belongs to a component, it describes the search state (Results, no results or waiting).

And the component is dispatching changes, I don't understand the second part of your comment.